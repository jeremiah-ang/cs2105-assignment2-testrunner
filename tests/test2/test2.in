hello
bye
see you again!
/send Alice.java
hellow
bye
see you again! 
/send Bob.java