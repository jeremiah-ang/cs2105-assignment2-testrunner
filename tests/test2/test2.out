hello
bye
see you again!
hellow
bye
see you again! 
